MotorControl/MotorControl/Resources/Kinetic_Tremor.json

```diff
@@ -0,0 +1,174 @@
+                "permissions":[
+                               {
+                                "permissionType":"motion",
+                                "reason":"Access to Motion and Fitness sensors is required to measure the phone's steadiness in your hand."
+                               }
+                            ],
+                 "image":{
+                     "type":"fetchable",
+                     "imageName":"FingerToNose9",
+                     "placementType":"topBackground"
+                 },
+                "videoResourceName": "instruction_video",
+                "videoBundleIdentifier": "org.sagebionetworks.MotorControl",
+                "icons": [
+                          {
+                            "icon":"SpaceToMoveYourArmsRose",
+                            "title": "SPACE TO MOVE YOUR ARMS"
+                          },
+                          {
+                            "icon":"ComfortablePlaceToSitRose",
+                            "title": "COMFORTABLE PLACE TO SIT"
+                          }
+                        ],
+                "actions":{
+                    "goForward":{
+                        "type":"default",
+                        "buttonTitle":"Get Started"
+                    },
+                    "skip":{
+                        "type":"reminder",
+                        "reminderIdentifier":"Kinetic Tremor"
+                    }
+                 }
+             },
+             {
+                 "identifier": "handSelection",
+                 "type": "handSelection",
+                 "title": "Which hands are you capable of doing this task with?",
+                 "inputFields":[{
+                     "uiHint": "list",
+                     "type": "singleChoice.string",
+                     "choices": [
+                         {
+                             "text": "I can only perform this activity with my LEFT hand.",
+                             "value": "left"
+                         },
+                         {
+                             "text": "I can only perform this activity with my RIGHT hand.",
+                             "value": "right"
+                         },
+                         {
+                             "text": "I can perform this activity with both hands.",
+                             "value": "both"
+                         }
+                     ]
+                 }]
+             },
+             {
+                "identifier":"eachHandInstructions",
+                "type":"instruction",
+                "title":"Each hand for 30 seconds",
```

Comment:
I was just following the example of the resting tremor which has this exact flow.  I feel like since it's never been a problem on any other app, we should just leave it as is.

Also, say we did move it in front of the which hand question... if they selected to only do it with one hand, that copy is still inaccurate.  It's unavoidable without custom dynamic copy which I think is overkill to fix a non-issue.

Reply:
If you look at the text for this view, it talks about "you will do this with your right hand and then with your left hand." Seems to me that it makes more sense to put it before asking about which hand, yeah?

Reply:
Ok, switched it to before the hand question step for you.